src/extension/inline/Transition.re

```diff
@@ -0,0 +1,167 @@
+type styleT;
+
+external makeStyle :
+ azimuth::string? =>
+ background::string? =>
+ backgroundAttachment::string? =>
+ backgroundColor::string? =>
+ backgroundImage::string? =>
+ backgroundPosition::string? =>
+ backgroundRepeat::string? =>
+ border::string? =>
+ borderCollapse::string? =>
+ borderColor::string? =>
+ borderSpacing::string? =>
+ borderStyle::string? =>
+ borderTop::string? =>
+ borderRight::string? =>
+ borderBottom::string? =>
+ borderLeft::string? =>
+ borderTopColor::string? =>
+ borderRightColor::string? =>
+ borderBottomColor::string? =>
+ borderLeftColor::string? =>
+ borderTopStyle::string? =>
+ borderRightStyle::string? =>
+ borderBottomStyle::string? =>
+ borderLeftStyle::string? =>
+ borderTopWidth::string? =>
+ borderRightWidth::string? =>
+ borderBottomWidth::string? =>
+ borderLeftWidth::string? =>
+ borderWidth::string? =>
+ bottom::string? =>
+ captionSide::string? =>
+ clear::string? =>
+ clip::string? =>
+ color::string? =>
+ content::string? =>
+ counterIncrement::string? =>
+ counterReset::string? =>
+ cue::string? =>
+ cueAfter::string? =>
+ cueBefore::string? =>
+ cursor::string? =>
+ direction::string? =>
+ display::string? =>
+ elevation::string? =>
+ emptyCells::string? =>
+ cssFloat::string? =>
+ font::string? =>
+ fontFamily::string? =>
+ fontSize::string? =>
+ fontSizeAdjust::string? =>
+ fontStretch::string? =>
+ fontStyle::string? =>
+ fontVariant::string? =>
+ fontWeight::string? =>
+ height::string? =>
+ left::string? =>
+ letterSpacing::string? =>
+ lineHeight::string? =>
+ listStyle::string? =>
+ listStyleImage::string? =>
+ listStylePosition::string? =>
+ listStyleType::string? =>
+ margin::string? =>
+ marginTop::string? =>
+ marginRight::string? =>
+ marginBottom::string? =>
+ marginLeft::string? =>
+ markerOffset::string? =>
+ marks::string? =>
+ maxHeight::string? =>
+ maxWidth::string? =>
+ minHeight::string? =>
+ minWidth::string? =>
+ orphans::string? =>
+ outline::string? =>
+ outlineColor::string? =>
+ outlineStyle::string? =>
+ outlineWidth::string? =>
+ overflow::string? =>
+ padding::string? =>
+ paddingTop::string? =>
+ paddingRight::string? =>
+ paddingBottom::string? =>
+ paddingLeft::string? =>
+ page::string? =>
+ pageBreakAfter::string? =>
+ pageBreakBefore::string? =>
+ pageBreakInside::string? =>
+ pause::string? =>
+ pauseAfter::string? =>
+ pauseBefore::string? =>
+ pitch::string? =>
+ pitchRange::string? =>
+ playDuring::string? =>
+ position::string? =>
+ quotes::string? =>
+ richness::string? =>
```

Comment:
lol, it's the CSS2Properties from the DOM spec (https://www.w3.org/TR/DOM-Level-2-Style/css.html#CSS-CSS2Properties) + a few CSS3 properties

The idea is to flesh it out and get it into rehydrate. Just stored here temporarily :)